lib/wor/paginate/utils/preserve_records_helper.rb

```diff
@@ -0,0 +1,64 @@
+require_relative 'uri_helper'
+
+module Wor
+  module Paginate
+    module Utils
+      class PreserveRecordsHelper
+        def initialize(content, url, options)
+          @content = content
+          @url = url
+          @options = options
+        end
+
+        def call
+          [content.where("#{field} <= :last_value", last_value: last_value),
+           UriHelper.replace_query_params(url, query_param_name => last_value)]
+        end
+
+        private
+
+        attr_reader :content, :url, :options
+
+        def by
+          @by ||= begin
+            by = options[:by]&.to_sym || :timestamp
```

Comment:
`timestamp` is created_at, or updated_at? :confused:

Reply:
Defaults to created_at but its configurable